src/components/PostalCodeInput.vue

```diff
@@ -3,34 +3,29 @@
     <label :for="id">
       {{label}}<span v-if="isRequiredAsteriskShown" class="required-asterisk">*</span>
     </label><br/>
-    <masked-input
+    <input
         :id="id"
         type="text"
         name="postalCode"
         class="form-control"
         :data-cy="getCypressValue()"
-        :value="value"
-        :mask="mask"
-        :guide="false"
-        placeholderChar="#"
+        :value="modelValue"
+        v-maska="{ mask: 'A#A #A#'}"
         ref="input"
-        @input="inputHandler($event)"
+        @input.stop="inputHandler($event)"
```

Comment:
Hey, just rethinking stopping the base input event. I added this on the facility number input because I didn't want it to fire 2 input events, once with the full event and once with just the value, I thought it could lead to weird race condition bugs. I am just thinking if that was correct though, it might be the case that some app is depending on the full event to be included in an `input` event. Do you know if that would be the case anywhere?

Reply:
I'm not sure if the `.stop` is necessary. As long as the component's `input` event is triggered with the value passed, it shouldn't break existing functionality.

Reply:
I think the stop is necessary actually because I noticed during the upgrade that the `$event` argument on became an event object rather than just the value (which is why I had to add that `const value` assignment in the `inputHandler` function. If we don't explicitly emit the `event.target.value` then just the `event` object would be emitted and cause weird behaviors, no?

Reply:
Oh that makes sense, so in vue 2 it didn't even emit with the full event. Then having `.stop` I agree is the right approach, I'll put that in on the digitInput as well. Might need to go back and check the merged in ones